### PR TITLE
docs(): Rewriting the cache page with the new informations and more details

### PR DESCRIPTION
**Pull request by asturur**

### CODE REVIEW

File: src/content/docs/docs/Old docs/fabric-object-caching.mdx [hunk 1]
```diff
@@ -5,111 +5,146 @@ title: Caching of object, properties and beahvior
...
+  /**
+   * Pixel limit for cache canvases. 1Mpx , 4Mpx should be fine.
+   * @since 1.7.14
+   * @type Number
+   * @default
+   */
+  perfLimitSizeTotal = 2097152;
+
+  /**
+   * Pixel limit for cache canvases width or height. IE fixes the maximum at 5000
```
Comment (Smrtnyk) [inline src/content/docs/docs/Old docs/fabric-object-caching.mdx:110]: Is IE even relevant now,  I am not sure if someone even uses it now, or if can use it now at all

Comment (tiesont) [reply src/content/docs/docs/Old docs/fabric-object-caching.mdx:110]: > Is IE even relevant now, I am not sure if someone even uses it now, or if can use it now at all

IE is [officially unsupported by Microsoft](https://support.microsoft.com/en-us/windows/internet-explorer-downloads-d49e1f0d-571c-9a7b-d97e-be248806ca70), at least for "public" users. 

> Support for Internet Explorer ended on June 15, 2022. Internet Explorer 11 has been permanently disabled through a Microsoft Edge update on certain versions of Windows 10.
>
> If any site you visit needs Internet Explorer 11, you can reload it with Internet Explorer mode in Microsoft Edge.

There _are_ still businesses and government entities which pay for continued support for Windows 7, but.... probably not a browser the Fabric team really needs to be concerned about. Those same entities can pay for someone to handle IE quirks with respect to Fabric.js, if they really need it, at least IMHO.

Comment (asturur) [reply src/content/docs/docs/Old docs/fabric-object-caching.mdx:110]: Internet explorer is unsupported in fabric 6, that note can be reworded to remeber why we chose a number under 5000 as default value.

I m still heavily editing this page

**Comment by asturur** [comment]
i'll have to re-read in a couple of days but i m merging as is.
There is a lot of new information and details that i think are good anyway

File: src/content/docs/docs/Old docs/fabric-object-caching.mdx [hunk 2]
```diff
@@ -5,111 +5,146 @@ title: Caching of object, properties and beahvior
...
+   * Lowest pixel limit for cache canvases, set at 256PX
+   * @since 1.7.14
+   * @type Number
+   * @default
+   */
+  minCacheSideLimit = 256;
+```
+
+minCacheSideLimit will determine the mini size of a side of the cache. At any object size the cache canvas will be at least 256 by 256 pixels.
+If you are caching a Rect that is 1 by 1, you will have a 256 by 156 canvas with a single pixel filled.
```
Comment (drinkspiller) [inline src/content/docs/docs/Old docs/fabric-object-caching.mdx]: >  256 by 156

Should "256 by 156" be "256 by 256"?

Comment (asturur) [reply src/content/docs/docs/Old docs/fabric-object-caching.mdx]: it is 256 by 256

**Pull request merged**